packages/acceptance-tests/README.md

```diff
@@ -0,0 +1,64 @@
+# HubSpot CLI Acceptance Tests
```

Comment:
FWIW, it is possible to configure `lerna` to find packages outside `packages` which might be a good move for `acceptance-tests` since it is not public and has a different purpose than the packages in `packages/*`.

Reply:
I did [set the package as private](https://github.com/HubSpot/hubspot-cli/pull/466/files#diff-96d7c7c218525b55bef08e7648ed7b79f8e161db3c88763cbff609f9935a2208R5) so it won't be published. Do we want to also move the package elsewhere? If so, where do you want it moved?

Reply:
I think it would be better at the top-level as `acceptance-tests` and added to the `packages` section of the [lerna config](https://github.com/lerna/lerna#lernajson) as separate entry without a glob.

Reply:
This would have the added benefit that `jest` would be scoped to `packages` and the jasmine usage would be specific to the acceptance-tests.

Reply:
I moved the `acceptance-tests` folder out of the `packages` directory and restored the jest config package settings to look at `packages/*` again.